command_callbacks.py

```diff
@@ -1754,6 +1754,12 @@ def savestate_filename(input_dict, environment_dict):
                            environment_dict['cmdargs'], environment_dict['defaulttarget'], 
                            environment_dict['defaultkeyname'], environment_dict['autosave'], 
                            environment_dict['currentkeyname'])
+  except TypeError:
+    raise seash_exceptions.UserError("Please load at least a pubkey, and work as that identity.")
+    
+  except UnboundLocalError:
```

Comment:
This (and the `TypeError` above) feel odd to check against, even if that is what `seash_helper` currently raises.

Can we make `seash_helper` raise more appropriate exceptions? How do its other commands deal with situations like these?